ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -1133,3 +1133,33 @@ def test_jax_numpy_arcsinh(
         fn_tree="numpy.arcsinh",
         x=x[0],
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("bool"),
```

Comment:
The test should also provide cases for integral inputs, not just booleans.

Reply:
Can you please elaborate on what do you mean for integral inputs?

Reply:
integer arrays.